src/angular/3-creating-components/angular.json

```diff
@@ -21,7 +24,8 @@
             "index": "src/index.html",
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
-            "tsConfig": "src/tsconfig.app.json",
+            "tsConfig": "tsconfig.app.json",
```

Comment:
can we use eslint instead - and add a section about that?

Reply:
I think we can, but I'd rather do it after I update all pages to angular 13, since eslint is not a default configuration yet.

Reply:
created issue https://bitovi.atlassian.net/browse/NGACAD-31 for later